repair/row_level.cc

```diff
@@ -764,10 +764,9 @@ class repair_meta {
         });
     }
 
-    static std::unordered_map<node_repair_meta_id, lw_shared_ptr<repair_meta>>& repair_meta_map() {
-        static thread_local std::unordered_map<node_repair_meta_id, lw_shared_ptr<repair_meta>> _repair_metas;
```

Comment:
Very happy to see this static move out the method, working with function statics is a major pain in gdb. It has a very weird syntax that is so easy to get wrong (or forget) and gdb often forgets the type, and spelling out correctly a template (with all the hidden defaults) is one of the hardest things to do in gdb.

Reply:
Yes. That is why I am moving this out. It is painful to work with it!

Reply:
btw, sometimes, thread_local is not working. so, I am considering do not use the thread_local for the repair_meta objects in debug space.

Reply:
How do you plan on avoiding `thread_local`? We can't have all threads use the same list, that will result in corruption.

I think you can rely on `thread_local` working, it is such a fundamental building block in our code that without thread debugging there is really no debugging at all.

Reply:
std::vector<std::list<*repair_meta>> metas, metas[x] can only be accessed on shard x.